inspectors/models/notation/timesignature/timesignaturesettingsmodel.cpp

```diff
@@ -0,0 +1,53 @@
+#include "timesignaturesettingsmodel.h"
+
+TimeSignatureSettingsModel::TimeSignatureSettingsModel(QObject* parent, IElementRepositoryService* repository) :
+    AbstractInspectorModel(parent, repository)
+{
+    setModelType(TYPE_TIMESIGNATURE);
+    setTitle(tr("Time signatures"));
+    createProperties();
+}
+
+void TimeSignatureSettingsModel::createProperties()
+{
+    m_hasToShowCourtesy = buildPropertyItem(Ms::Pid::SHOW_COURTESY);
```

Comment:
`hasToShowCourtesy`? Why not just `showCourtesy`?

Reply:
Consistency with [keysignaturesettingsmodel.cpp](https://github.com/vpereverzev/MuseScore/blob/0dd6aa0ba73f93802548802c6cdbe5d2efea171c/inspectors/models/notation/keysignature/keysignaturesettingsmodel.cpp#L13), but I agree that `showCourtesy` is better assuming this a a simple `on`/`off` switch rather than `auto`/`force` which "hasTo" would imply.

Reply:
"is" and "has" are usually markers that it's a bool property
In QML, "showCourtesy" may sound like a direct action - i.e. "Q_INVOKABLE void showCourtesy()"